Code/Framework/AzToolsFramework/AzToolsFramework/AssetDatabase/AssetDatabaseConnection.cpp

```diff
@@ -413,6 +427,22 @@ namespace AzToolsFramework
                     SqlParam<const char*>(":sourcename"),
                     SqlParam<const char*>(":platform"));
 
+            static const char* QUERY_PRODUCT_BY_SOURCENAME_SCANFOLDERID_PLATFORM = "AzToolsFramework::AssetDatabase::QueryProductBySourceNameScanFolderIDPlatform";
+            static const char* QUERY_PRODUCT_BY_SOURCENAME_SCANFOLDERID_PLATFORM_STATEMENT =
+                "SELECT Products.*, Jobs.* FROM Sources LEFT JOIN Jobs "
+                                                                                "ON Sources.SourceID = Jobs.SourcePK INNER JOIN Products "
+                                                                                "ON Jobs.JobID = Products.JobPK WHERE "
+                                                                                "Jobs.Platform = :platform AND "
+                                                                                "Sources.SourceName = :sourcename AND Sources.ScanFolderPK = :scanfolderid;";
```

Comment:
why are these lines tabbed so far over to the right?

Reply:
Auto formatter which does whatever it wants... Ill update it.